docs/pages/app-signing/app-credentials.mdx

```diff
@@ -48,7 +48,7 @@ This certificate will be used for all of your apps. If this certificate expires,
 
 Apple Push Notification Keys (often abbreviated as APN keys) allow the associated apps to send and receive push notifications.
 
-You can have a maximum of 2 APN keys associated with your Apple Developer account, and a single key can be used with any number of apps. If you revoke an APN key, all apps that rely on that key will no longer be able to send or receive push notifications until you upload a new key to replace it. Uploading a new APN key **will not** change your users' [Expo Push Tokens](/versions/latest/sdk/notifications#notificationsgetexpopushtokenasync). Push notification keys do not expire. You can clear the APN key Expo currently has stored for your app by running `eas credentials` and following the prompts.
+You can have a maximum of 2 APN keys associated with your Apple Developer account, and a single key can be used with any number of apps. If you revoke an APN key, all apps that rely on that key will no longer be able to send or receive push notifications until you upload a new key to replace it. Uploading a new APN key **will not** change your users' [Expo Push Tokens](/versions/latest/sdk/notifications#notificationsgetexpopushtokenasync). Push notification keys do not expire. You can clear the APN key Expo currently has stored for your app by running `eas credentials` and following the prompts. Expo-created APN keys can be downloaded for your organization via [expo.dev](https://expo.dev).
```

Comment:
Hello @RRaideRR, thank you for the contribution! Adding a not about that is a good idea. 👍 

```suggestion
You can have a maximum of 2 APN keys associated with your Apple Developer account, and a single key can be used with any number of apps. If you revoke an APN key, all apps that rely on that key will no longer be able to send or receive push notifications until you upload a new key to replace it. Uploading a new APN key **will not** change your users' [Expo Push Tokens](/versions/latest/sdk/notifications#notificationsgetexpopushtokenasync). Push notification keys do not expire. You can clear the APN key Expo currently has stored for your app by running `eas credentials` and following the prompts. 

> APN keys created by Expo can be downloaded on the [Expo website](https://expo.dev/accounts/[account]/projects/[project]/credentials/ios).
```

A rephrase suggestion + updated website link utilizing account resolver.